staging/src/k8s.io/api/apps/v1beta2/types.go

```diff
@@ -29,6 +29,191 @@ const (
 	StatefulSetRevisionLabel       = ControllerRevisionHashLabelKey
 )
 
+type DaemonSetUpdateStrategy struct {
```

Comment:
You need to mark these new APIs WIP (see other structs in this file) as we're going to break it. Please put DaemonSet APIs in the roughly same location as extensions/v1beta1 (i.e. after Deployment).